README.md

```diff
@@ -1,14 +1,41 @@
 
 
 # Project Name
+Hello Microverse Project!
 
 > One paragraph statement about the project.
+This is my first project in microverse
 
 
+# Built With
+Html
+Css
+
 
 Additional description about the project and its features.
 
-My first project at Microverse.
+The project is a welcome page, welcoming anyone to microverse.
+
+👤 Author
+
+GitHub: @Anny85-code
+
+Twitter: @Annyudo8
+
+LinkedIn: https://www.linkedin.com/in/aniekan-udo
```

Comment:
[OPTIONAL]

- [ ] I know you want your Readme to be professional, so I'm here to help you achieve that. Look at the images below and try having all your social media handle look like that. Bet you will like the outcome. 😄 

implementation:

<img width="254" alt="chrome_cCeMpAGd2U" src="https://user-images.githubusercontent.com/81468675/143137917-392810c4-cc0d-452e-b1fd-cd689e9f2ffa.png">

result:
<img width="107" alt="chrome_WZqPxVaqCj" src="https://user-images.githubusercontent.com/81468675/143138057-2480a9c7-5bd0-4558-bd09-ffb6145948a5.png">